src/pages/DatasetRegistration.js

```diff
@@ -16,34 +16,25 @@ import * as fp from 'lodash/fp';
 import './DataAccessRequestApplication.css';
 
 
-const noOptionMessage = 'Start typing a Dataset Name, Sample Collection ID, or PI';
+// const noOptionMessage = 'Start typing a Dataset Name, Sample Collection ID, or PI';
```

Comment:
Think this is part of the broken TypeOfResearch component, I'm still troubleshooting atm but that's not necessary for dataset submission